common/src/lib.rs

```diff
@@ -95,8 +89,17 @@ pub struct BlockHeader {
     pub tx_merkle_root: Hash256,
     /// The Merkle root of the non-essential state.
     pub state_merkle_root: Hash256,
-    /// The essential state.
-    pub essential_state: EssentialState,
+    /// The effective net validator set (delegation-applied) for the next block.
+    /// The order here is the leader order.
```

Comment:
Is it same with leader selection priority order? (mentioned in [line 23](https://github.com/postech-dao/simperby/pull/60/files#diff-d38884f7391ae50f795707dbb281656e8afe38ea3ecac15a6ee80d52724017c0R23))

Reply:
yes, and added some desc.